config/generate.go

```diff
@@ -56,17 +61,49 @@ func GenerateConfig() error {
 		return err
 	}
 
-	return writeConfig(i)
+	if isMissionControlUIPresent {
+		// Ask for the admin username
+		err = survey.AskOne(&survey.Input{Message: "Mission Control (UserName)", Default: "admin"}, &i.AdminName, survey.Required)
+		if err != nil {
+			return err
+		}
+
+		// Ask for the admin password
+		err = survey.AskOne(&survey.Input{Message: "Mission Control (Password)", Default: "admin123"}, &i.AdminPass, survey.Required)
+		if err != nil {
+			return err
+		}
+
+		// Ask for the admin role
+		err = survey.AskOne(&survey.Input{Message: "Mission Control (Role)", Default: "super-admin"}, &i.AdminRole, survey.Required)
+		if err != nil {
+			return err
+		}
+
+		i.HomeDir = utils.UserHomeDir()
+		i.BuildVersion = utils.BuildVersion
+	}
+
+	if configFilePath == "none" {
+		configFilePath = "./" + i.ID + ".yaml"
```

Comment:
Use the path separator from the os package

Reply:
Let's avoid using `./` completely

Reply:
Completed